model/Security/Vocabularies/AssessmentType.md

```diff
@@ -0,0 +1,25 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# AssessmentType
+
+## Summary
+
+Specifies the type of assessment data a hasAssessmentFor relationship contains.
+
+## Description
+
+Vulnerabilities and elements linked with a hasAssessmentFor express a security
+assessment. The type of assessment must be noted on the assessmentType property
+of a relationship subclassed from VulnAssessmentRelationship.
+
+## Metadata
+
+- name: AssessmentType
+
+## Entries
+
+- cvss2: Common Vulnerability Scoring System v2
+- cvss3: Common Vulnerability Scoring System v3
+- exploitCatalog: Known exploits catalog
+- ssvc: Stakeholder-Specific Vulnerability Categorization
+- vex: Vulnerability Exploitability eXchange
```

Comment:
I think `epss` is missing from this list to pair with https://github.com/spdx/spdx-3-model/blob/main/model/Security/Classes/EpssVulnAssessmentRelationship.md?

Reply:
epss: Exploit Prediction Scoring System

Reply:
Good catch Rose, thanks.
Fixed :+1: